Fix FFI definition `Py_Exit` (never returns, was `()` return value, now `!`).
